wiki/Guides/Using_custom_hitsounds/de.md

```diff
@@ -1,49 +1,68 @@
 
-Wie `normal-hitclap2.wav` oder wie `soft-hitfinish3.wav`
+## Eigene Hitsounds anwenden
 
-Man sollte nicht vergessen, dass im *Timing Setup* der Hitsound auf Custom mit der jeweiligen Nummer gestellt werden muss.
+![](img/adding-custom-hitsounds.png "osu! anweisen, wie eigene Hitsounds zu verwenden sind")
 
-![Tutorial Image 2](img/adding-custom-hitsounds.png "Tutorial Image 2")
+Damit osu! deine eigenen Hitsounds korrekt abspielt, stelle sicher dass du die zweite Option, "Custom 1", wie im oberen Bild gezeigt ausgewählt hast. Standardmäßig werden eigene Samplesets als `<SS>:C1` abgekürzt, wobei `<SS>` der erste Buchstabe der Sampleset-Gruppe ist. Also entweder N (Normal), S (Soft), oder D (Drum).
+
+Beachte, dass du  nicht jeden Ton im Sampleset mit einem eigenen Ton ersetzen musst. Im ersten Bild wird dir auffallen, dass keine Datei namens `soft-slidertick.wav` vorhanden ist. In diesem Fall wird osu den Standardton für alle regulären Slidertick-Treffer verwenden, wenn das "Soft"-Sampleset verwendet wird.
+
+## Mit mehreren eigenen Hitsound-Sets arbeiten
+
+Manchmal beinhaltet ein Song mehrere verschiedene Musikstile und eine Gruppe von Hitsounds wird nicht zu allen passen. In diesem Fall ist es oft hilfreich, gänzlich andere Hitsounds (oder Hitsound-Gruppen) zu verwenden. Das kann durch das Hinzufügen einer Zahl an das Ende des Dateinamens eines Hitsounds erreicht werden:
+
+`<sampleset>-<ton><#>.wav`
+
+Hier kann `<#>` jede Zahl deiner Wahl sein. Der osu!-Editor unterstützt von Haus aus Zahlen zwischen 2 und 100. Höhere Zahlen können, falls notwendig, durch Bearbeitung der `.osu`-Datei erreicht werden. Beachte, dass die erste Gruppe an Hitsounds nicht zwangsläufig die Zahl "1" verwenden muss, auch wenn mehrere Hitsound-Gruppen verwendet werden. Demnach wird `soft-hitclap1.wav` nicht funktionieren. Stattdessen wird `soft-hitclap.wav` verwendet werden.
+
+Um sicherzustellen, dass die verschieden bezifferten Hitsounds oder Hitsound-Gruppen richtig abgespielt werden, wirst du einen geerbten Timing-Point (grüne Linie) hinzufügen und das Sampleset von "Custom 1" wie unten gezeigt auf die Option darunter setzen müssen. Hier kannst du die Zahl der Hitsound-Gruppe eintragen, die du verwenden willst.
+
+![](img/using-multiple-hitsound-sets.png "Zu einer zweiten eigenen Hitsound-Gruppe wechseln")
+
+Wenn das Sampleset eines geerbten Timing-Points wie oben gezeigt auf `S:C2` gesetzt ist, werden die Standard-Hitsounds und Hitsound-Unterteilungen mit den entsprechend benannten eigenen Hitsounds ersetzt, z. B. `soft-hitclap2.wav`, falls diese vorhanden sind. Diese werden solange verwendet werden, bis ein vererbter Timing-Point mit einem anderen Sampleset erreicht wird - in diesem Bild `02:00:723`, wo das Sampleset wieder auf `S:C1` gewechselt wird.
```

Comment:
```suggestion
Wenn das Sampleset eines geerbten Timing-Points wie oben gezeigt auf `S:C2` gesetzt ist, werden die Standard-Hitsounds und Hitsound-Unterteilungen mit den entsprechend benannten eigenen Hitsounds ersetzt, z. B. `soft-hitclap2.wav`, falls diese vorhanden sind. Diese werden solange verwendet werden, bis ein vererbter Timing-Point mit einem anderen Sampleset erreicht wird — in diesem Bild `02:00:723`, wo das Sampleset wieder auf `S:C1` gewechselt wird.
```
Just a personal preference of mine: I intend to use em dashes from the original article to reduce required changes when translating phrases. You may still choose to use hyphens.

Relevant: [ASC/Writing § Gedankenstriche](https://osu.ppy.sh/wiki/de/Article_styling_criteria/Writing#gedankenstriche)